draft-ietf-core-groupcomm-bis.md

```diff
@@ -297,6 +297,33 @@ When this happens, the client normally processes at the CoAP layer each of those
 
 Then, the application is in a better position to decide what to do, depending on the available context information. For instance, it might accept and process all the responses from the same server, even if they are not Observe notifications (i.e., they do not include an Observe option). Alternatively, the application might accept and process only one of those responses, such as the most recent one from that server, e.g. when this can trigger a change of state.
 
+## Caching ## {#sec-caching}
+
+CoAP endpoints that are members of a CoAP group MAY cache responses as intended in Section 5.6 of {{RFC7252}}. In particular, the same rules apply for the set of request options used as "Cache-Key".
+
+Furthermore, building on what defined in Section 8.2.1 of {{RFC7252}}:
+
+   * A client sending a GET or FETCH group request over multicast MAY update a cache with the responses from the servers in the CoAP group. Then, the client uses both cached-still-fresh and new responses as the result of the group request.
+   
+   * A client sending a GET or FETCH group request over multicast MAY use a response received from a server to satisfy a subsequent request sent intended to that server on the related unicast request URI. The unicast request URI is obtained by replacing the authority part of the request URI with the transport-layer source address of the response message.
+
+   * A cache MAY revalidate a response by making a GET or FETCH request on the
+   related unicast request URI.
+   
+Note that, in the presence of proxies, this requires the client to distinguish the different responses to the same group request, as well as the different origin servers. This in turn requires additional means to provide the client with information about the origin server of each response, as discussed in {{sec-proxy-caching}}.
```

Comment:
"this requires" can be clarified - all bullets are a MAY here so does not really seem a requirement?
It is only needed when the client wants to do one of the optional (MAY) things in the bullets. We can also leave the text as is and I'll think about improved formulation.